src/wcm.cpp

```diff
@@ -794,10 +797,12 @@ BindingsDynamicList::BindingWidget::BindingWidget(const std::string & cmd_name,
     command_label.set_size_request(OPTION_LABEL_SIZE);
     command_label.set_alignment(Gtk::ALIGN_START);
     command_box.pack_start(command_label, false, false);
-    expander.set_label("Command " + cmd_name);
+    expander.set_label(fmt::format(_("Command {name}"), fmt::arg("name", cmd_name)));
```

Comment:
Same here
I generally prefer `fmt` (`std`) too, but IMHO since we are stuck with Gtk in the project we should use its utilities and not add dependencies.

Reply:
@NamorNiradnug I think @Secret-chest is right here, and the dependencies added are available in most distributions and don't require a particular (latest) version. It might be nice if the dependencies were opt-in and just don't do translations if a dependency is not found but I expect this would make the code more complicated and open the door for wcm builds without translation support. So, it should be ok to hard-dep on these so we can make sure folks can 'just' install wcm, set LANG and be on their way.

Reply:
> But the GLib methods don't allow changing the order of placeholders, meaning they won't work in many languages

Why? It supports placeholders of form `%i`, where `i` is a digit. Such a placeholder is be substituted with the `i`th argument. E.g. `Glib::ustring::compose(_("Command %1"), cmd_name)`.

Since soreau is ok about adding the dependency, I won't insist on this change, but please consider it.